resources/views/inputs/input-with-prefix.blade.php

```diff
@@ -15,8 +15,10 @@
             ])
         @endunless
 
-        <div
-            class="input-wrapper input-wrapper-with-prefix @error($name) input-text--error @enderror"
+        <div @class([
+                'input-wrapper input-wrapper-with-prefix',
+                'input-text--error' => $errors->has('name'),
```

Comment:
```suggestion
                'input-text--error' => $errors->has($name),
```

Reply:
Will have a check now, but I think we can also do something like `'input-text--error' => @error($name)` too 🤔 Any preference if that one works ?

Reply:
Ok no it doesn't :trollface: